From 40abff4062d0521cf2b2a8a4660a3d2933f86f9d Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Mon, 1 Apr 2024 09:09:07 +0000
Subject: [PATCH] Add acinclude.m4 to include required macros

* These micro files are in the m4 directories, but the aclocal didn't
  add them to aclocal.m4 automatically, so add them to acinclude.m4
  manually.

* The runlog.m4 is added for python.m4.

Upstream-Status: Inappropriate [OE Specific]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 acinclude.m4 |  8 ++++++++
 m4/runlog.m4 | 17 +++++++++++++++++
 2 files changed, 25 insertions(+)
 create mode 100644 acinclude.m4
 create mode 100644 m4/runlog.m4

diff --git a/acinclude.m4 b/acinclude.m4
new file mode 100644
index 0000000000..118b7f0e5f
--- /dev/null
+++ b/acinclude.m4
@@ -0,0 +1,8 @@
+m4_include([m4/fr_init_module.m4])
+m4_include([m4/ax_with_prog.m4])
+m4_include([m4/runlog.m4])
+m4_include([m4/python.m4])
+m4_include([m4/ax_compare_version.m4])
+m4_include([m4/libcurl_check_config.m4])
+m4_include([m4/ax_ruby_devel.m4])
+m4_include([m4/ax_prog_ruby_version.m4])
diff --git a/m4/runlog.m4 b/m4/runlog.m4
new file mode 100644
index 0000000000..690efc3258
--- /dev/null
+++ b/m4/runlog.m4
@@ -0,0 +1,17 @@
+##                                                          -*- Autoconf -*-
+# Copyright (C) 2001-2018 Free Software Foundation, Inc.
+#
+# This file is free software; the Free Software Foundation
+# gives unlimited permission to copy and/or distribute it,
+# with or without modifications, as long as this notice is preserved.
+
+# AM_RUN_LOG(COMMAND)
+# -------------------
+# Run COMMAND, save the exit status in ac_status, and log it.
+# (This has been adapted from Autoconf's _AC_RUN_LOG macro.)
+AC_DEFUN([AM_RUN_LOG],
+[{ echo "$as_me:$LINENO: $1" >&AS_MESSAGE_LOG_FD
+   ($1) >&AS_MESSAGE_LOG_FD 2>&AS_MESSAGE_LOG_FD
+   ac_status=$?
+   echo "$as_me:$LINENO: \$? = $ac_status" >&AS_MESSAGE_LOG_FD
+   (exit $ac_status); }])
-- 
2.35.5

